defaults.lua

```diff
@@ -224,22 +224,23 @@ FRONTLIGHT_SENSITIVITY_DECREASE = 2
 -- insensitive sort
 DALPHA_SORT_CASE_INSENSITIVE = true
 
+-- no longer needed
```

Comment:
luacheck is probably complaining because it keeps a list of those globals to ignore, right? ;).

Reply:
Didn't check what you mean by complaining, but if something like `SEARCH_LIBRARY_PATH` is removed here it should also be removed from .luacheckrc to make sure that it would complain.

Reply:
Also `spec/unit/defaults_spec.lua` needs to be updated. I will take a look tonight.